src/UI/Buttons/MobileSubmit.module.css

```diff
@@ -1,6 +1,5 @@
 .mobile {
 	position: unset;
-	z-index: 2;
```

Comment:
Deze had een z-index die niks deed?
(ff dubbel checken vandaar deze comment)

Reply:
Aaah ja in het ticket had ik dit beschreven.

> In theorie all z-indexes could be removed accept for Chat and Launcher (tested) but their might be stuff in v1 that is not in v2 that would require this) i did went true the index and check what would be logical for z-index and rearranged the numbering.

Ze konden overal weg behalve chat en launcher. Maar dacht misschien voor features die nog niet in v2 zitten laat ik het er maar in staan. Vond de verzend knop dan wat overdreven om die er in te laten als bij actions (div erboven) al een hogere z-index zat.

Reply:
Als blijkt dat de z-index niet nodig is mag deze van mij helemaal weg. We hoeven geen rekening te houden met dingen die we in de toekomst evt toe gaan voegen waar een z-index voor nodig is. Als we dat gaan bouwen zullen we daar de z-index gaan toevoegen/aanpassen indien nodig.

Reply:
Resolved, alle onnodige z-indexes zijn nu weg